task/04-arrays-tasks.js

```diff
@@ -353,7 +354,7 @@ function getItemsSum(arr) {
  *  [ null, undefined, NaN, false, 0, '' ]  => 6
  */
 function getFalsyValuesCount(arr) {
-   throw new Error('Not implemented');
+   return arr.length - arr.filter(i => !!i).length;
```

Comment:
in you solution using `!!i` is redundant 
`i` will be implicitly converted into Boolean `return arr.length - arr.filter(i => i).length;`
But better `return arr.filter(i => !i).length;`